### PR TITLE
fix(examples): satisfy new Model interface

### PR DESCRIPTION
**Pull request by bashbunni**
## Changes

- [x] update examples to use the new Model API (they weren't running before)
- [x] update `go.mod` to use bubbles from the `beta` branch
- [x] update tests to run, though they're not passing. 

## Still TODO (can be done in another PR to limit scope)

- [ ] make `TestClearMsg` pass
- [ ] make `TestTeaWithFilter` not hang

## Questions

> Is there anything else context should be used for in these examples?

### CODE REVIEW

File: examples/table-resize/main.go [hunk 1]
```diff
@@ -31,13 +31,12 @@ func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
...
 
-func (m model) View() string {
+func (m model) View(ctx tea.Context) string {
 	return "\n" + m.table.String() + "\n"
 }
 
 func main() {
-	re := lipgloss.NewRenderer(os.Stdout)
-	baseStyle := re.NewStyle().Padding(0, 1)
+	baseStyle := lipgloss.NewStyle().Padding(0, 1)
```
Comment (aymanbagabas) [inline examples/table-resize/main.go:39]: The table initialization and creation should be inside `Init(ctx)` to be able to use `ctx.NewStyle()`

File: examples/tabs/main.go [hunk 2]
```diff
@@ -55,7 +55,7 @@ var (
 	windowStyle       = lipgloss.NewStyle().BorderForeground(highlightColor).Padding(2, 0).Align(lipgloss.Center).Border(lipgloss.NormalBorder()).UnsetBorderTop()
```
Comment (aymanbagabas) [inline examples/tabs/main.go:55]: These styles should be moved to the model and use `ctx.NewStyle()` instead

File: tutorials/basics/main.go [hunk 3]
```diff
@@ -24,11 +24,11 @@ func initialModel() model {
 	}
 }
 
-func (m model) Init() tea.Cmd {
+func (m model) Init(ctx tea.Context) (tea.Model, tea.Cmd) {
 	return tea.SetWindowTitle("Grocery List")
```
Comment (aymanbagabas) [inline tutorials/basics/main.go:28]:
```suggestion
	return m, tea.SetWindowTitle("Grocery List")
```

File: tutorials/basics/README.md [hunk 4]
```diff
@@ -76,7 +76,7 @@ some initial I/O. For now, we don't need to do any I/O, so for the command,
 we'll just return `nil`, which translates to "no command."
 
 ```go
-func (m model) Init() tea.Cmd {
+func (m model) Init(ctx tea.Context) (tea.Model, tea.Cmd) {
     // Just return `nil`, which means "no I/O right now, please."
     return nil
```
Comment (aymanbagabas) [inline tutorials/basics/README.md:81]:
```suggestion
    return m, nil
```

File: tutorials/commands/README.md [hunk 5]
```diff
@@ -92,7 +92,7 @@ Note that we don't call the function; the Bubble Tea runtime will do that when
 the time is right.
 
 ```go
-func (m model) Init() (tea.Cmd) {
+func (m model) Init(ctx tea.Context) (tea.Model, tea.Cmd) {
     return checkServer
```
Comment (aymanbagabas) [inline tutorials/commands/README.md:96]:
```suggestion
    return m, checkServer
```

File: tutorials/commands/main.go [hunk 6]
```diff
@@ -35,11 +35,11 @@ type errMsg struct{ err error }
 // error interface on the message.
 func (e errMsg) Error() string { return e.err.Error() }
 
-func (m model) Init() tea.Cmd {
+func (m model) Init(ctx tea.Context) (tea.Model, tea.Cmd) {
 	return checkServer
```
Comment (aymanbagabas) [inline tutorials/commands/main.go:39]:
```suggestion
	return m, checkServer
```

**Comment by bashbunni** [comment]
Still need to update pipe example, but all the others are good to go

**Pull request closed without merging**